rdmd.d

```diff
@@ -916,3 +895,58 @@ void yap(size_t line = __LINE__, T...)(auto ref T stuff)
     debug stderr.writeln(line, ": ", stuff);
     else stderr.writeln(stuff);
 }
+
+//
+// std.file wrappers that also log their operations
+//
+auto exists(R)(R name)
```

Comment:
Is there any need for the templates here?  It's easier to just support `string` and modify if another use-case ever comes up.

Reply:
templates are easier for me